### PR TITLE
Subscription information section

### LINKED ISSUE #1390: Display Subscription management instead of plans if the user is already subscribed
If the user is subscribed, a few options should be displayed instead of showing the available plans. The two options that must be shown are: 

- Pause payment 
- Cancel subscription 

Below is a screenshot example of the look and feel: 

subscription

### PR DESCRIPTION
**Pull request by RodriSanchez1**
close #1390

### CODE REVIEW

File: src/components/Settings/Subscribe/Subscribe.component.js [hunk 1]
```diff
@@ -100,61 +101,70 @@ const Subscribe = ({
...
+                      <br />
+                      <br />
+                      <FormattedMessage {...messages.includedFeatures} />
+                    </Typography>
+                    <List disablePadding style={{ padding: '5px' }}>
+                      {renderIncludedFeatures()}
+                    </List>
+                  </CardContent>
+                  <CardActions>
+                    <Button size="small">Learn More</Button>
```
Comment (tomivm) [inline src/components/Settings/Subscribe/Subscribe.component.js:159]: there is an empty action to Learn more button

File: src/components/Settings/Subscribe/Subscribe.component.js [hunk 2]
```diff
@@ -100,61 +101,70 @@ const Subscribe = ({
...
+                key={offer.id}
+                item
+                xs={12}
+                sm={6}
+                style={{ padding: '5px', maxWidth: 328 }}
+              >
+                <Card style={{ minWidth: 275 }} variant="outlined">
+                  <CardContent>
+                    <Typography
+                      sx={{ fontSize: 19 }}
```
Comment (tomivm) [inline src/components/Settings/Subscribe/Subscribe.component.js:128]: sx is not available on MUI V4

File: src/components/Settings/Subscribe/SubscriptionInfo.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    <Paper elevation={3} className="Subscribe__Info">
+      <Typography variant="h5">
+        <FormattedMessage {...messages.subscriptionInfo} />
+      </Typography>
+      <div className="Subscribe__Info__Container">
+        <div className="Subscribe__Info__Table__Container">
+          <Table aria-label="simple table">
+            <TableBody>
+              {Object.entries(subscription).map(row => (
+                <TableRow key={row[0]}>
```
Comment (tomivm) [inline src/components/Settings/Subscribe/SubscriptionInfo.js]: You could use a constant to replace the magic number '0'

File: src/components/Settings/Subscribe/SubscriptionInfo.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                  <TableCell align="right">
+                    {row[0] === 'status' ? (
+                      <Chip
+                        label={<FormattedMessage {...messages[row[1]]} />}
+                        size="small"
+                        color="primary"
+                        style={statusColor}
+                      />
+                    ) : (
+                      row[1]
```
Comment (tomivm) [inline src/components/Settings/Subscribe/SubscriptionInfo.js]: you could replace the magic number '1'

**Review by tomivm — COMMENTED** [review]
Pretty component!

File: src/components/Settings/Subscribe/Subscribe.component.js (hunk 1 above)
Comment (RodriSanchez1) [reply src/components/Settings/Subscribe/Subscribe.component.js:159]: will fix on #1423

File: src/components/Settings/Subscribe/Subscribe.component.js (hunk 2 above)
Comment (RodriSanchez1) [reply src/components/Settings/Subscribe/Subscribe.component.js:128]: will fix on #1423

**Pull request merged**